tests/test.py

```diff
@@ -154,6 +154,8 @@
 TEST_VAULT_TLS_CA = abs_path(os.environ.get('TEST_VAULT_TLS_CA', 'tests/ssl/ca/ca.crt'))
 VAULT_KV_ENGINE_VERSION=os.environ.get('VAULT_KV_ENGINE_VERSION', 'v1')
 CRYPTO_ENVELOPE_HEADER = b'%%%'
+TLS_CERT_CLIENT_ID_1 = None
```

Comment:
please, add comment where this variables should be initialized (`setupModule` function). now I used ctrl+f to understand is it temporal stub during refactoring or new variables that used somewhere)

Reply:
Ok, will do that